examples/scripts/sparsecode_poisson_pgm.py

```diff
@@ -146,10 +196,10 @@ def plot_results(hist, str_ss, L0, xsol, xgt, Amat):
 
 """
 Use default PGMStepSize object, set L0 based on norm of Forward
-operator and set up AcceleratedPGM solver object. Run the solver and
+operator and set up AcceleratedPGM solver object.  Run the solver and
```

Comment:
Let's not re-introduce double spacing after a period after my lengthy campaign to remove them! (See other examples below too.)

Reply:
Not sure why the diff shows that I changed this. I didn't.

Actually, there still seem to be many instances of this in other files. I'll create a separate issue to get this fixed.

Reply:
Oh, I think it is because of the file name changes that the diff gets confused. I'll change the file name back for now and we can change it again later.

Reply:
Isn't it better just to live with the confused diff?

Reply:
In the confused diff the new BlockArray example is compared to the (deleted) non-BlockArray example. Confusing ;)

Reply:
Undoing the renaming didn't fix it. Once you rename your dog "cat" and then give it cat food (GitHub line comments) it apparently will always think it is a cat.